coopr-docs/docs/source/installation/packages.rst

```diff
@@ -162,20 +162,20 @@ RPM using Yum
 -------------
 Download the Cask Yum repo definition file::
 
-  sudo curl -o /etc/yum.repos.d/cask.repo http://repository.cask.co/downloads/centos/6/x86_64/cask.repo
+  sudo curl -o /etc/yum.repos.d/coopr.repo http://repository.cask.co/centos/6/x86_64/coopr/0.9/coopr.repo
```

Comment:
Do you want 0.9 hardcoded, or is this a version that should be a replace and change with each release?